test/certs/generate.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+cd "$(realpath -- $(dirname -- "$0"))"
+
+ipki() {
+  # Minica generates everything in-place, so we need to cd into the subdirectory.
+  mkdir ipki
+  cd ipki
+
+  # Check that `minica` is installed
+  command -v minica >/dev/null 2>&1 || {
```

Comment:
This command checking chunk should come out of `ipki` and go to line 5.

Reply:
It only needs to happen if the ipki directory needs to be regenerated. If it doesn't need to be regenerated, then we don't need minica to be installed. And the `set -e` at the top guarantees that this function exiting with a non-zero status will cause the whole script to exit with a non-zero status as well.

But I have moved it above the creation of the ipki directory, so that if the check fails, we haven't created an empty dir that will cause the next execution to skip it.